src/App.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import logo from './images/logo.png';
+import * as style from './styles/style.css';
+
+export default class App extends React.Component {
+
+    render() {
```

Comment:
add please `editorconfig`. In one place you have 4-identation tab, in other files 2-identation